templates/commands/goldentest/verify.go

```diff
@@ -204,18 +220,18 @@ func (c *VerifyCommand) Run(ctx context.Context, args []string) (rErr error) {
 
 		if outputMismatch {
 			failureText := red(fmt.Sprintf("golden test [%s] didn't match actual output, you might "+
-				"need to run 'record' command to capture it as the new expected output", tc.TestName))
+				"need to run 'record' command to capture it as the new expected output for template location %s", tc.TestName, templateLocation))
```

Comment:
I think this would be a little nicer putting the template location at the beginning of the string next to the test name. Like:

`fmt.Sprintf("template [%s] golden test [%s] didn't match.......`

Here and elsewhere in this PR.